operatorcourier/identify.py

```diff
@@ -28,7 +28,7 @@ def get_operator_artifact_type(operatorArtifactString):
                     artifact_type = operatorArtifact["kind"]
                     artifact_name = operatorArtifact['metadata']['name']
             if artifact_type is not None:
-                logger.info('Parsed %s: %s', artifact_type, artifact_name)
+                logger.info('Parsed {}: {}'.format(artifact_type, artifact_name))
```

Comment:
No please. This is correct. Logger should have lazy evaluation.

https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation

Reply:
Understood, I will update the error and warning methods to allow lazy evaluation